docs/feature-gates.md

```diff
@@ -250,8 +245,26 @@ metadata:
     ipam.antrea.io/ippools: 'pool1'
 ```
 
+#### Data path change for this feature
+
+When `AntreaIPAM` is enabled, `antrea-agent` will connect the Node's network interface
+to the OVS bridge at startup, and it will detach the interface from the OVS bridge and
+restore its configurations at exit. Node may lose network connection when `antrea-agent`
+or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
+`AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
+
+All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
+destination MAC is rewritten to the Pod's MAC address, including `AntreaIPAM` Pods and
+regular `Subnet per Node` IPAM Pods, even they are not in the same subnets. Inter-Node
+traffic will be sent from source Node to destination Node directly.
+
 #### Requirements for this Feature
 
-This feature is currently supported on Linux Nodes only. Annotation of single IP pool
-is supported. In future, annotation of up to two pools of different IP versions will be
-supported.
+In Antrea 1.4, this feature is supported on Linux Nodes, with IPv4, `system` OVS datapath
+type, and `noEncap`, `noSNAT` traffic mode.
+
+The IPs in the `IPPools` must be in the same "underlay" subnet as the Node IP, because
+cross-Node traffic of AntreaIPAM Pods is sent to the Node network from the source Node
```

Comment:
As this is described earlier, we can save some words here, like  "because inter-Node traffic of `AntreaIPAM` Pods is forwarded by the Node network".

Reply:
Thanks. Fixed.